### PR TITLE
roachtest: disable AC in OR tests

### PR DESCRIPTION
**Pull request by dt**
Release note: none.
Epic: none.

### CODE REVIEW

**Review by stevendanna — COMMENTED** [review]
Could we add a comment either in the code or the commit message about why we are disabling this?  Might also be worth an issue reference.

**Comment by dt** [comment]
>  add a comment either in the code

Done.

**Comment by dt** [comment]
Actually, lamme scribble up some comments on the other settings here too while I'm in the neighborhood, one sec.

**Review by stevendanna — APPROVED** [review]
Thanks, very helpful

File: pkg/cmd/roachtest/tests/online_restore.go [hunk 1]
```diff
@@ -96,15 +96,48 @@ func registerOnlineRestore(r registry.Registry) {
...
+
+							// TODO(dt): what's the right value for this? How do we tune this
+							// on the fly automatically during the restore instead of by-hand?
+							// Context: We expect many operations to take longer than usual
+							// when some or all of the data they touch is remote. For now this
+							// is being blanket set to 1h manually, and a user's run-book
+							// would need to do this by hand before an online restore and
+							// reset it manually after, but ideally the queues would be aware
+							// of remote-ness when they pick their own timeouts and pick
+							// according.
```
[the remark this answers is not in the document]
Comment (dt) [reply pkg/cmd/roachtest/tests/online_restore.go]: Done.

**Comment by dt** [comment]
TFTR!

bors r+

**Pull request merged**